docs/sources/tempo/configuration/_index.md

```diff
@@ -390,14 +473,23 @@ metrics_generator:
         # The maximum length of label values. Label values exceeding this limit will be truncated.
         [max_label_value_length: <int> | default = 2048]
 
+    # Configuration block for the Write Ahead Log (WAL)
+    traces_storage: <WAL config>
+
+      # Path to store the WAL files.
+      # Must be set.
+      # Example: "/var/tempo/generator/traces"
+      [path: <string> | default = ""]
+
     # Storage and remote write configuration
     storage:
 
         # Path to store the WAL. Each tenant will be stored in its own subdirectory.
         path: <string>
 
         # Configuration for the Prometheus Agent WAL
-        wal:
+        # https://github.com/prometheus/prometheus/blob/v2.51.2/tsdb/agent/db.go#L62-L84
+        wal: <prometeus agent WAL config>
```

Comment:
is this typo for prometheus?

Reply:
Yes. 
I did a general spell check.
Tell me if you find anything else!

Reply:
Weird.
GH seems to be stuck with processing the change in this PR.
Will probs fix itself later.
For no just check the branch directly